app/models/manageiq/providers/ibm_power_vc/cloud_manager/template.rb

```diff
@@ -1,4 +1,9 @@
 ManageIQ::Providers::Openstack::CloudManager::Template.include(ActsAsStiLeafClass)
 
 class ManageIQ::Providers::IbmPowerVc::CloudManager::Template < ManageIQ::Providers::Openstack::CloudManager::Template
+  has_and_belongs_to_many :cloud_tenants,
+                          :foreign_key             => "vm_id",
+                          :join_table              => "cloud_tenants_vms",
+                          :association_foreign_key => "cloud_tenant_id",
+                          :class_name              => "ManageIQ::Providers::IbmPowerVc::CloudManager::CloudTenant"
```

Comment:
The Openstack subclass had `:class-name => "ManageIQ::Providers::Openstack::CloudManager::CloudTenant` which was causing saving the template to fail because it was trying to find the tenant by the wrong class name